src/header.c

```diff
@@ -62,7 +65,14 @@
 int
 read_wim_header(WIMStruct *wim, struct wim_header *hdr)
 {
+
+#ifdef _MSC_VER
```

Comment:
Alternatively, you could have the following in `compiler.h`:
```C
#ifdef _MSC_VER
#define PRAGMA_ALIGN(x, a) __declspec(align(a)) x
#else
#define PRAGMA_ALIGN(x, a)   x __attribute__((aligned(a)))
#endif
```
And then just simplify to:
```C
PRAGMA_ALIGN(struct wim_header_disk disk_hdr, 8);
```